### PR TITLE
JENKINS-55787 Switch labels from entry to checkbox

### PR DESCRIPTION
**Pull request by timja**
## [JENKINS-55787](https://issues.jenkins-ci.org/browse/JENKINS-55787) - Switch labels from entry to checkbox

<details>
<summary>2.227-SNAPSHOT</summary>

image

</details>

<details>
<summary>2.138.4</summary>

image

</details>

You can also test via reverting my revert jenkins core version commit

ref: https://github.com/jenkinsci/jenkins/pull/3895, cc @MarkEWaite @jsoref 

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/jenkinsci/git-plugin/blob/master/CONTRIBUTING.adoc) doc
- [x] I have referenced the Jira issue related to my changes in one or more commit messages
- [x] Unit tests pass locally with my changes
- [x] I have added documentation as necessary
- [x] No Javadoc warnings were introduced with my changes
- [x] No spotbugs warnings were introduced with my changes
- [x] I have interactively tested my changes

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -27,7 +27,7 @@
   <properties>
     <revision>4.3.0</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.138.4</jenkins.version>
+    <jenkins.version>2.227-SNAPSHOT</jenkins.version>
```
Comment (fqueiruga) [inline pom.xml]: Isn't this too much of a jenkins version bump? Wasn't it supposed to be 2.204?

FTR I know nothing of Maven and have a really basic idea how to read POM files, so I'm probably misunderstanding something

Comment (timja) [reply pom.xml]: That was just to test the UI out, I left the commit in if anyone else wants to test it, the last commit reverts this

**Comment by jsoref** [comment]
This supercedes #676

**Comment by romenrg** [comment]
I'm not very familiar with this, but here are my two cents:

Just tested this locally and it seems to work fine, although my UI looks slightly different to what I see in the PR description by @timja (I'm probably missing something):

Screen Shot 2020-04-05 at 11 41 14

From @jsoref comment and linked PR I understand the idea is to have attributes associated to the checkbox rather than to the entry.

Taking a look at the resulting html, I see this code:
Screen Shot 2020-04-05 at 11 48 05

Which seems to come from this change:
Screen Shot 2020-04-05 at 11 51 07

I wonder if this is the expected result. Maybe it is, just not sure, but I still see the title associated to the label instead of it being associated to the checkbox. Or maybe I'm doing something wrong.

Also, in the source code I still see the "field" attribute associated to the entry, while in Joe's PR I see the "field" attributes switched from the entry to the checkbox, along with all the others. Is there a reason to keep the field attribute in the entry in these cases?

**Comment by timja** [comment]
There’s been a few commits to the PR since the screenshot was taken, help icon alignment has been fixed and related nested item layout has been adjusted

That would be why it looks different

**Pull request merged**